backend/api/views/response_rates.py

```diff
@@ -0,0 +1,65 @@
+from api.core import logger
+from api.models import db, CensusResponse
+import re
+import time
+
+
+def get_year(date):
+    return str(time.strptime(date, "%m%d%Y").tm_year)
+
+'''
+returns a collection of CensusResponses
+filtered by tract_id if tract_id parameter is not None
+'''
+def get_census_responses(tract_id, state):
+    if state:
+        regex = re.compile("^{}.*".format(state))
```

Comment:
pls comment what this does for readability

Reply:
yup my bad added it back